packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -538,31 +549,96 @@ class Lines extends StatelessWidget {
         final lineNum = index + 1;
         return LineItem(
           lineContents: lines[index],
+          debuggerController: debuggerController,
           pausedFrame: pausedLine == lineNum ? pausedFrame : null,
         );
       },
     );
   }
 }
 
-class LineItem extends StatelessWidget {
+class LineItem extends StatefulWidget {
   const LineItem({
     Key key,
     @required this.lineContents,
+    @required this.debuggerController,
     this.pausedFrame,
   }) : super(key: key);
 
+  static const _hoverDelay = Duration(milliseconds: 500);
```

Comment:
we already have a const for this in common_widgets (tooltipWait)

Reply:
actually, seeing how this is used, using tooltipWait may not be as clear. fine to leave.